ddp-workspace/projects/ddp-sdk/src/lib/components/validationMessage.component.ts

```diff
@@ -11,7 +11,7 @@ import * as _ from 'underscore';
             </li>
         </ul>
         <ng-template #singleMessage>
-            <span [innerHTML]="(isArrayOfMessages() ? message[0] : message) | translate" routeTransformer></span>
```

Comment:
The compiler was not happy with this line: the translate pipe can handle only strings, but the compiler assumes that `message` can be a string or array of strings.